tests/test_ec2/test_spot_fleet.py

```diff
@@ -64,10 +64,19 @@ def spot_config(subnet_id, allocation_strategy="lowestPrice"):
                 "EbsOptimized": False,
                 "WeightedCapacity": 4.0,
                 "SpotPrice": "10.00",
+                "TagSpecifications": [
+                    {
+                        "ResourceType": "instance",
+                        "Tags": [{"Key": "test", "Value": "value"}],
+                    }
+                ],
             },
         ],
         "AllocationStrategy": allocation_strategy,
         "FulfilledCapacity": 6,
+        "TagSpecifications": [
+            {"ResourceType": "instance", "Tags": [{"Key": "test2", "Value": "value2"}]}
```

Comment:
From the AWS docs:

> The key-value pair for tagging the Spot Fleet request on creation. The value for `ResourceType` must be `spot-fleet-request`, otherwise the Spot Fleet request fails.

Reply:
@bpandola 
Thank you for your reviews!
Should we check if `ResourceType` is correct or not here?
``` python
# moto/ec2/models/spot_requests.py
self.tags: Dict[str, Any] = (
    convert_tag_spec(tag_specifications)
    if tag_specifications is not None
    else {}
)
```